x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -20,16 +20,35 @@ import {
   TrustedApp,
 } from '../../../../../common/endpoint/types';
 import { HttpFetchOptions } from 'kibana/public';
-import { TRUSTED_APPS_LIST_API } from '../../../../../common/endpoint/constants';
+import {
+  TRUSTED_APPS_GET_API,
+  TRUSTED_APPS_LIST_API,
+} from '../../../../../common/endpoint/constants';
+import {
+  GetPackagePoliciesResponse,
+  PACKAGE_POLICY_API_ROUTES,
+} from '../../../../../../fleet/common';
+import { EndpointDocGenerator } from '../../../../../common/endpoint/generate_data';
+import { isFailedResourceState, isLoadedResourceState } from '../state';
+import { forceHTMLElementOffsetWidth } from './components/effected_policy_select/test_utils';
+import { resolvePathVariables } from '../service/utils';
+import { toUpdateTrustedApp } from '../../../../../common/endpoint/service/trusted_apps/to_update_trusted_app';
+import { useIsExperimentalFeatureEnabled } from '../../../../common/hooks/use_experimental_features';
 
 jest.mock('@elastic/eui/lib/services/accessibility/html_id_generator', () => ({
   htmlIdGenerator: () => () => 'mockId',
 }));
 
+// TODO: remove this mock when feature flag is removed
```

Comment:
I'm wondering if we can avoid this by instead just dispatching an action to the store that sets the value to what we want it to be before executing the tests (ex. in a `beforeEach()`)

Reply:
You mean creating a new action and dispatching it from the tests? Sounds strange to me have an action on the store only for testing purposes.

Reply:
Ahh. Yes, right now given the current implementation, it does not make sense - it would be a test only action. I'm ok to leave it as is. 

_(FYI: the kibana "config" also exposes its self as an Observable that you can listen to for changes and then react to them. I was thinking in that setup we would have an action to dispatch the env config to the store)_